src/Settings.tsx

```diff
@@ -91,29 +118,42 @@ const Settings: React.FC<SettingsProps> = (props) => {
         const selectedAudioDevice = audioDevices?.find((device) => {
             return device.deviceId === selectedMic
         });
- 
+
         videoStream?.getTracks().forEach(track => track.stop())
-        await props.onSubmit(selectedVideoDevice, selectedAudioDevice);
+
+        let turnConfig = undefined;
+        if (turnServerEnable) {
+
+            turnConfig = {
+                urls: turnServerURL,
+                username: turnServerUsername,
+                credential: turnServerCredential
+            } as RTCIceServer;
+
+        }
+        const iceTransportPolicy = (turnServerOnlyRelay ? 'relay' : 'all') as RTCIceTransportPolicy;
```

Comment:
Should we initialize iceTransportPolicy as undefined like turnConfig and only set it to a value if turnServerEnable is true? It would be strange to still persist a value if turnServerEnable is false